### PR TITLE
Update drone

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

**Comment by 6543** [comment]
go vet will work if #41 got merged

**Comment by 6543** [comment]
@lunny rebase/update ;)

**Comment by 6543** [comment]
ok now vet works ... but golint find a lot to fix ... :sweat_smile:

**Comment by lunny** [comment]
Let's disable lint at first. :(

**Comment by 6543** [comment]
for (newfile) L73:
`go test -mod=vendor -cover -coverprofile coverage.out $(PACKAGES) || exit 1`

File: Makefile [hunk 1]
```diff
@@ -58,15 +58,19 @@ vet:
...
+		export BINARY="golangci-lint"; \
+		curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin v1.31.0; \
 	fi
-	for PKG in $(PACKAGES); do golint -set_exit_status $$PKG || exit 1; done;
+	golangci-lint run --timeout 5m
 
 .PHONY: test
 test:
-	for PKG in $(PACKAGES); do go test -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $$PKG || exit 1; done;
+	for PKG in $(PACKAGES); do go test -mod=vendor -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $$PKG || exit 1; done;
```
Comment (6543) [inline Makefile]:
```suggestion
	go test -mod=vendor -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $(PACKAGES) || exit 1
```

File: Makefile [hunk 2]
```diff
@@ -58,15 +58,19 @@ vet:
...
+		export BINARY="golangci-lint"; \
+		curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin v1.31.0; \
 	fi
-	for PKG in $(PACKAGES); do golint -set_exit_status $$PKG || exit 1; done;
+	golangci-lint run --timeout 5m
 
 .PHONY: test
 test:
-	for PKG in $(PACKAGES); do go test -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $$PKG || exit 1; done;
+	go test -mod=vendor -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $(PACKAGES) || exit 1
```
Comment (6543) [inline Makefile]:
```suggestion
	go test -mod=vendor -cover -coverprofile coverage.out $(PACKAGES) || exit 1
```

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
ping

**Review by zeripath — APPROVED** [review]

**Pull request merged**